src/modules/python/pbs/v1/_pmi_none.py

```diff
@@ -1,6 +1,6 @@
 
 """
-/* 
```

Comment:
Why are there C comment /* */ in a python file?

Reply:
I have removed C comments from all the python file headers.